docker/driver.go

```diff
@@ -0,0 +1,96 @@
+package docker
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/bblfsh/performance"
+
+	errors "gopkg.in/src-d/go-errors.v1"
+	"gopkg.in/src-d/go-log.v1"
+)
+
+// One day it will become a config file but now it's to annoying to ship it
+var drivers = map[string]string{
+	"bash":       "bash-driver",
+	"cpp":        "cpp-driver",
+	"csharp":     "csharp-driver",
+	"go":         "go-driver",
+	"java":       "java-driver",
+	"javascript": "javascript-driver",
+	"php":        "php-driver",
+	"python":     "python-driver",
+	"ruby":       "ruby-driver",
+	"typescript": "typescript-driver",
+}
+
+// InstallDriver downloads driver's repository, checkouts to a given commit,
+// builds docker image and installs driver's image to bblfshd container.
+// Requires bblfshd container running.
+func InstallDriver(language, commit string) error {
+	if err := validateLanguage(language); err != nil {
+		return err
+	}
+
+	driver := drivers[language]
+	image := driverImage(driver, commit)
+	if err := downloadAndBuildDriver(driver, commit, image); err != nil {
+		return err
+	}
+	if err := installDriverToBblfshd(bblfshdContainer, language, image); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func downloadAndBuildDriver(driver, commit, image string) error {
+	tmpDir := filepath.Join(os.TempDir(), driver)
```

Comment:
```go
tmpDir, err := ioutil.TempDir("", driver)
if err != nil {
   return err
}
log.Debugf("Created temp directory %v", tmpDir)
…
```

Reply:
Should we clean this up when we're done? Does the temp directory need to outlive the build step?

Reply:
1st part is Done.

2nd - it's a good question that refers to the main purpose of this CLI:
- running in Jenkins: then post-cleanup is not necessary
- local debug: necessary
Done. Added closer to related functions